src/SFA.DAS.FindEmploymentSchemes.Contentful.ContentCodeGenerator/Program.cs

```diff
@@ -83,115 +70,64 @@ private static async Task GenerateSchemesContent(ContentfulClient client, HtmlRe
-                Console.WriteLine($"{await AsHtmlString(scheme.Offer, htmlRenderer)},");
-                Console.WriteLine($"{await AsHtmlString(scheme.AdditionalFooter, htmlRenderer)}");
-                Console.WriteLine("),");
+                Console.WriteLine($"            new {typeName}(\"{scheme.Name}\",");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.ShortDescription)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.ShortCost)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.ShortBenefits)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.ShortTime)},");
+                Console.WriteLine($"                \"{scheme.Url}\", {scheme.Size},");
+
+                Console.Write("             new string[] {");
+                Console.Write($"                {string.Join(", ", scheme.FilterAspects.Select(f => $"\"{f}\""))}");
+                Console.WriteLine("             },");
+
+                Console.WriteLine($"                {GenerateHtmlString(scheme.DetailsPageOverride)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.Description)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.Cost)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.Responsibility)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.Benefits)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.CaseStudies)},");
+                Console.WriteLine($"                \"{scheme.OfferHeader}\",");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.Offer)},");
+                Console.WriteLine($"                {GenerateHtmlString(scheme.AdditionalFooter)}");
+                Console.WriteLine("                ),");
             }
 
             Console.WriteLine(@"        };");
         }
 
-        private static HtmlRenderer CreateHtmlRenderer()
-        {
-            var htmlRendererOptions = new HtmlRendererOptions
-            {
-                ListItemOptions =
-                {
-                    OmitParagraphTagsInsideListItems = true
-                }
-            };
-            var htmlRenderer = new HtmlRenderer(htmlRendererOptions);
-            htmlRenderer.AddRenderer(new GdsCtaContentRenderer(htmlRenderer.Renderers));
-            htmlRenderer.AddRenderer(new GdsHeadingRenderer(htmlRenderer.Renderers));
-            htmlRenderer.AddRenderer(new GdsHorizontalRulerContentRenderer());
-            htmlRenderer.AddRenderer(new GdsHyperlinkContentRenderer(htmlRenderer.Renderers));
-            htmlRenderer.AddRenderer(new GdsListContentRenderer(htmlRenderer.Renderers));
-            htmlRenderer.AddRenderer(new GdsParagraphRenderer(htmlRenderer.Renderers));
-            return htmlRenderer;
```

Comment:
The embedded Youtube content renderer has disappeared

Reply:
The YouTube renderer isn't in main yet.

Reply:
Ah, you're right. Although the new renderer code definitely won't merge successfully with these changes (because they upgrade to the new version of contentful), so it might be easier to address it here.